app/components/employeeCard.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import styles from "./employeeCard.module.scss";
+
+export default function EmployeeCard(props: {
+  employeeName: string;
+  employeeRole: string;
+  employeeImageSrc: string;
+}) {
+  return (
+    <div className={styles["employeeCard"]}>
```

Comment:
Also without having seen the design. I assume each card would be placed in a list? 
For a11y we'd want a `<ul />` of `<ol />` and assign  number of items with each item as `<li />`. Not sure if `EmployeeCard` would be wrapped in a list item later?